csuntechlab/iSTART
# STR-72 -- FE: Clicking on modules in the modules page causes the hashtag to appear

# Description
Added hashbang to routes; updated router-link for navbar
  
# Testing
1. Ensure login now has a hashbang *_`#`_*.
2. Click the navbar and click the menu items and ensure it contains only a single hashbang.
